subsys/mgmt/mcumgr/transport/src/smp_udp.c

```diff
@@ -60,13 +74,13 @@ struct configs {
 static struct configs configs = {
 #ifdef CONFIG_MCUMGR_TRANSPORT_UDP_IPV4
 	.ipv4 = {
-		.proto = "IPv4",
+		.proto = PROTOCOL_IPV4,
 		.sock  = -1,
```

Comment:
`.sock` is initialised and used by the thread
```suggestion
```

Reply:
As above, if the thread is started and stopped before the network interface comes up, the value needs to be a known value to prevent closing an invalid socket